util/remove-packages.sh

```diff
@@ -3,14 +3,8 @@
 source $(dirname $0)/packages.env
```

Comment:
I'm not sure if this script has any purpose/value anymore so feel free to remove it :)